src/app_service/service/keycard/service.nim

```diff
@@ -163,20 +163,17 @@ QtObject:
       debug "keycardStartFlow", kcServiceCurrFlow=($self.currentFlow), payload=payload, response=response
 
   proc resumeFlow(self: Service, payload: JsonNode) =
-    if self.busy:
-      return
```

Comment:
do u know why this was added in the first place? could it be it prevent other flow to work properly?